QCircLearning/trainerModel.py

```diff
@@ -113,10 +110,14 @@ def NN_opt(func, x0, callback=None, **kwargs):
                     f"Run ID: {kwargs['run_id']}, Iteration {iteration + 1}/{max_iter}"
                 )
                 sys.stdout.flush()
+            # start_time = time.time()
```

Comment:
Remove the time record part as they are irrelevant to Feature/early-stop

Reply:
okay